lte/gateway/c/core/oai/tasks/amf/Registration.cpp

```diff
@@ -691,6 +695,7 @@ int amf_send_registration_accept(amf_context_t* amf_context) {
 
 static int registration_accept_t3550_handler(zloop_t* loop, int timer_id,
                                              void* arg) {
+  OAILOG_FUNC_IN(LOG_AMF_APP);
```

Comment:
At line, https://github.com/magma/magma/pull/12348/files#diff-5971eeebd29f80bdbd0931787b020513ed6287d30b033723dbff1f8c65e4de37R706, module name LOG_NAS_AMF is used.
Could you include the relevant module.

Reply:
Comment addressed.